tests/system-tests/rdscore/internal/rdscorecommon/egress-ip.go

```diff
@@ -672,6 +681,40 @@ func getNodeForReboot(isIPv6 bool) (string, string, error) {
 	return "", "", fmt.Errorf("no egress IP address found in egressIP map")
 }
 
+func setMinAvailableToZeroForActivePDB() error {
+	By("Workaround for the node drain failure due to active PodDisruptionBudget")
+
+	allAvailablePDB, err := poddisruptionbudget.ListInAllNamespaces(APIClient)
+
+	if err != nil {
+		glog.Infof("Failed to list all available pod disruption budget due to %v", err)
+
+		return nil
```

Comment:
if there's an error listing PDBs in all the namespaces, why it's not returned outside?

Reply:
because if there is no pdb found we don't care